go-controller/pkg/ovn/egressip_test.go

```diff
@@ -11235,6 +11237,14 @@ var _ = ginkgo.Describe("OVN master EgressIP Operations", func() {
 					ObjectMeta: newEgressIPMeta(egressIPName),
 					Spec: egressipv1.EgressIPSpec{
 						EgressIPs: []string{egressIP},
+						PodSelector: metav1.LabelSelector{
+							MatchLabels: egressPodLabel,
+						},
+						NamespaceSelector: metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"name": egressNamespace.Name,
```

Comment:
will this actually match the namespace? I thought it is `kubernetes.io/metadata.name` and not just `name`? Can you double check that this EIP selects the pod

Reply:
guess it does since the LRPs are created later.